src/tests/ftest/container/snapshot.py

```diff
@@ -277,6 +296,33 @@ def test_snapshot_negativecases(self):
                       "taking snapshot with a NULL epoch.")
 
 
+        #(6)DAOS-1392 Test snapshot with an invalid epoch
+        self.log.info(
+            "==(6)DAOS-1392 destroy snapshot with an invalid handle.")
+        try:
+            snapshot.destroy(None, epoch)
+            self.fail(
+                "##(6)Negative test destroy snapshot with an "
+                "invalid coh handle, expected fail, shown Passing##")
+        except Exception as error:
+            self.log.info(
+                "==>Negative test, destroy snapshot with an invalid handle.")
+            self.log.info("   Expected Error: %s", str(error))
+
+        #(7)DAOS-1388 Verify snap_list bad parameter behavior
+        self.log.info(
+            "==(7)DAOS-1388 Verify snap_list bad parameter behavior.")
+        try:
+            snapshot.list(None, 0)
+            self.fail(
+                "##(7)Negative test snapshot list with an "
+                "invalid coh and epoch, expected fail, shown Passing##")
+        except Exception as error:
```

Comment:
Same as above this and all other such cases in this test where test is expected to fail.

Reply:
Thanks for your comment.
Yes.
Snapshot negative test with invalid epoch or snapshot list handle will generate Error of RC: -1002.
Testlog:
==(6)DAOS-1392 destroy snapshot with an invalid handle.
==>Negative test, destroy snapshot with an invalid handle.
   Expected Error: Failed to destroy the snapshot. RC: -1002
==(7)DAOS-1388 Verify snap_list bad parameter behavior.
==>Negative test, snapshot list with an invalid coh.
   Expected Error: Snapshot create returned non-zero. RC: -1002

Reply:
Please see the new script with exact error verification.